skbio/alignment/_alignment.py

```diff
@@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
                                           "or fn is provided.")
 
         if ids is not None:
-            fn = lambda _: ids
```

Comment:
It's ridiculous ...

On (Feb-12-15| 9:32), Evan Bolyen wrote:
> > @@ -627,7 +627,9 @@ def update_ids(self, ids=None, fn=None, prefix=""):
> >                                            "or fn is provided.")
> >  
> >          if ids is not None:
> > -            fn = lambda _: ids
> 
> It would seem so.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/scikit-bio/pull/819/files#r24599822